content/d-sgov/d-sgov-právní-akty/d-sgov-právní-akty-glosář.ttl

```diff
@@ -166,10 +167,11 @@ d-sgov-právní-akty-pojem:fragment a skos:Concept;
   skos:prefLabel "Fragment"@cs .
 
 d-sgov-právní-akty-pojem:fragment-znění a skos:Concept;
+  skos:altLabel "Označení fragmentu znění právního aktu"@cs ;
   skos:definition """Fragment znění je označení části znění právního aktu. Příkladem je zákon/par/1/cast/4/.
 Navrhuji přejenovat na \"Vyjádření fragmentu znění právního aktu\""""@cs;
   skos:inScheme d-sgov-právní-akty:glosář;
-  skos:prefLabel "Označení fragmentu znění právního aktu"@cs .
+  skos:prefLabel "Fragment znění"@cs .
```

Comment:
Zase naopak. altLabel: "Fragment znění", prefLabel: "Označení fragmentu znění právního aktu". Je to tak, že prefLabel je to, co to je a altLabel je to, jak se tomu říká.

Reply:
+ koukám, že v tomto i předchozím případě by se s tím mělo změnit i IRI

Reply:
viz výše.